content/en/serverless/azure_functions/_index.md

```diff
@@ -0,0 +1,130 @@
+
+2. **Start the Datadog serverless compatibility layer and initialize the Node.js tracer**. Add the following lines to your main application entry point file (for example, `app.js`):
+
+   ```js
+   require('@datadog/serverless-compat').start();
+
+   // This line must come before importing any instrumented module. 
+   const tracer = require('dd-trace').init()
+   ```
+
+3. (Optional) **Enable runtime metrics**. See [Node.js Runtime Metrics][2].
+
+4. (Optional) **Enable custom metrics**. See [Metric Submission: DogStatsD][3].
+
+[1]: /tracing/trace_collection/automatic_instrumentation/?tab=singlestepinstrumentation
+[2]: /tracing/metrics/runtime_metrics/nodejs/?tab=environmentvariables
+[3]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=nodejs
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+1. **Install dependencies**. Run the following commands:
+   ```shell
+   pip install datadog-serverless-compat
+   pip install ddtrace
+   ```
+
+   To use [automatic instrumentation][1], you must use `dd-trace` v2.19+.
+
+   Datadog recommends using the latest versions of both `datadog-serverless-compat` and `ddtrace` to ensure you have access to enhancements and bug fixes.
+
+2. **Initialize the Datadog Python tracer and serverless compatibility layer**. Add the following lines to your main application entry point file:
+
+   ```python
+   from datadog_serverless_compat import start
+   from ddtrace import tracer, patch_all
+
+   start()
+   patch_all()
+   ```
+
+3. (Optional) **Enable runtime metrics**. See [Python Runtime Metrics][2].
+
+4. (Optional) **Enable custom metrics**. See [Metric Submission: DogStatsD][3].
+[1]: /tracing/trace_collection/automatic_instrumentation/?tab=singlestepinstrumentation
+[2]: /tracing/metrics/runtime_metrics/python/
+[3]: /metrics/custom_metrics/dogstatsd_metrics_submission/?code-lang=python
+{{< /programming-lang >}}
+{{< /programming-lang-wrapper >}}
+
+3. **Deploy your function**.
+
+4. **Configure Datadog intake**. Add the following environment variables to your function's application settings:
+   ```
+   DD_API_KEY=<YOUR_API_KEY>
+   DD_SITE={{< region-param key=dd_site code="true" >}}
+   ```
+   - Set `DD_API_KEY` to your [Datadog API key][1].
+   - Set `DD_SITE` to {{< region-param key=dd_site code="true" >}}. (Ensure that your correct [Datadog site][2] is selected in the drop-down on the right side of this page.)
+
+5. **Configure Unified Service Tagging**. You can collect metrics from your Azure Functions by installing the [Datadog Azure integration][6]. To correlate these metrics with your traces, first set the `env`, `service`, and `version` tags on your resource in Azure. Then, configure the following environment variables. You can add custom tags as `DD_TAGS`.
+
+    ```
+    DD_ENV="<ENVIRONMENT>"
```

Comment:
Just want to double check that these environment variables use quotation marks, but the ones in the previous step don't?